src/main/cli/cli.c

```diff
@@ -5030,15 +5030,19 @@ const cliResourceValue_t resourceTable[] = {
     DEFS( OWNER_LED_STRIP,     PG_LED_STRIP_CONFIG, ledStripConfig_t, ioTag ),
 #endif
 #ifdef USE_UART
-    DEFA( OWNER_SERIAL_TX,     PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagTx[0], SERIAL_PORT_MAX_INDEX ),
-    DEFA( OWNER_SERIAL_RX,     PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagRx[0], SERIAL_PORT_MAX_INDEX ),
+    DEFA( OWNER_SERIAL_TX,     PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagTx[0], SERIAL_PORT_MAX_INDEX - SERIAL_LPUART_MAX_INDEX),
+    DEFA( OWNER_SERIAL_RX,     PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagRx[0], SERIAL_PORT_MAX_INDEX - SERIAL_LPUART_MAX_INDEX),
```

Comment:
Isn't it better to define serialPinConfig_t { `ioTagTx[SERIAL_PORT_MAX_INDEX + SERIAL_LPUART_MAX_INDEX];` ... }

Reply:
Subtracting SERIAL_LPUART_MAX_INDEX (1) from SERIAL_PORT_MAX_INDEX (11) to have 10 UARTs and 1 LPUART.

Reply:
But you must change both, or it will break.

But a lot of hardwired problems will surface if you change it, so keeping change minimal may be better